README.md

```diff
@@ -1 +1,17 @@
-In progress!
+## This is a web application where you can find and view news, events and astronauts of different space agencies.
+
+Uses [TheSpaceDevs API](https://thespacedevs.com/llapi)
```

Comment:
1. Please add `The ssspace` to `readme` as the project name
2. `TheSpaceDevs` => `The Space Devs`
2. add info about `yarn build` command